hbase-server/src/test/java/org/apache/hadoop/hbase/master/assignment/TestRegionReplicaSplit.java

```diff
@@ -109,11 +112,16 @@ public void testRegionReplicaSplitRegionAssignment() throws Exception {
         }
       }
       // There are 6 regions before split, 9 regions after split.
-      HTU.getAdmin().split(table.getName(), Bytes.toBytes(1));
+      HTU.getAdmin().split(table.getName(), Bytes.toBytes("d"));
       int count = 0;
       while (true) {
         for (RegionServerThread rs : HTU.getMiniHBaseCluster().getRegionServerThreads()) {
           for (Region r : rs.getRegionServer().getRegions(table.getName())) {
+            // Make sure that every region has some data (even for split daughter regions).
+            if (RegionReplicaUtil.isDefaultReplica(r.getRegionInfo())) {
+              assertTrue(r.getStore(f).hasReferences() ||
```

Comment:
Why do you expect there to be data in all stores?

Reply:
That is per @busbey 's comments regarding with updating the split point, I think it is ok to make sure that after the split point change, every daughter region has some data, so coming up the assert. Since replica regions has some delay to map the data, the assert only makes sure that the primary(default) region gets data.